test/extended/images/mysql_replica.go

```diff
@@ -27,8 +27,8 @@ var (
 		{
 			"5.5",
 			"https://raw.githubusercontent.com/openshift/mysql/master/5.5/examples/replica/mysql_replica.json",
-			// TODO: Investigate why this is broken.
-			true,
+			// NOTE: Set to true in case of flakes.
```

Comment:
@mnagy is enabling the mysql replication tests with this change, not disabling it, so what's the follow up?